src/Raven.Server/Documents/Handlers/DocumentReplicationHandler.cs

```diff
@@ -15,13 +14,21 @@ namespace Raven.Server.Documents.Handlers
 {
     public class DocumentReplicationRequestHandler : DatabaseRequestHandler
     {
-        [RavenAction("/databases*/lastSentChangeVector", "GET", "@/databases/{databaseName: string}/lastSentChangeVector")]
-        public async Task GetLastSentChangeVector()
+        [RavenAction("/databases*/lastSentEtag", "GET", "@/databases/{databaseName: string}/lastSentEtag?srcDbId={databaseUniqueId:string}")]
+        public Task GetLastSentEtag()
```

Comment:
Why have this in the first place. We should be negotiating this _inside the websocket request_, not as a separate call.